### ISSUE TITLE
Unable to cast to chromecast device (nest hub)

### ISSUE THREAD

**Issue by balonchiks** [AlexxIT/WebRTC#9]
The component works great, and the video stream starts almost instantly, which is super cool! However, when trying to cast (using `service: cast.show_lovelace_view`) to my nest hub i get the `Custom element doesn't exist: webrtc-camera` issue. I know lots of cards has this issue as something extra has to be done by the developer to support casting of custom cards. It's mentioned here in the [Cast FAQ](https://cast.home-assistant.io/faq.html) at the very bottom, titled **Why do some custom cards not work?**

Do you mind taking a look at it please? That would be awesome if we can cast lovelace views with these super fast webrtc video streams!

**Comment by balonchiks** [comment]
ok, figured out myself. the issue is with the homeassistant cast component and HACS file locations. For some reason to make custom card work (at least this custom card) one need to copy the WebRTC component from `/custom_components/` (whcih is the default folder for HACS) to `/www/community/` and then add another line in configuration.yaml:
```
    - type: module
      url: /local/community/webrtc/www/webrtc-camera.js
```
which basically loads the component for the second time, but this is apparently the only way I was able to find to make the `Custom element doesn't exist: webrtc-camera` error disappear.

**Comment by AlexxIT** [comment]
I will check myself

**Comment by balonchiks** [comment]
sure, you might find [this link](https://community.home-assistant.io/t/home-assistant-cast-with-custom-cards/137457/14) interesting as there's been some discussion about this

**Comment by AlexxIT** [comment]
Fixed in v1.4.0, thanks for links
